app/models/galaxy_constructor.rb

```diff
@@ -19,7 +20,14 @@ def generate_system(ring)
     raw = rand(10) + rand(10) + 2
     angle = rand
     System.new({
-      name: name, census: 0, max_census: max_census, raw: raw, infrastructure: 0, ring: ring, angle: angle
+      name: name,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.